en/editor/components/spine.md

```diff
@@ -0,0 +1,234 @@
+
+2. Then right-click on the Spine node in the **Hierarchy** panel and select **Create -> UI Render -> Sprite** to add a child Sprite node (tail, hand and claw nodes in the image below) to it.
+
+    ![attach1](./spine/attach1.png)
+
+3. Select the Spine node in the **Hierarchy** panel for which you want to set the attachment slot, and add the slot's information to the child node by setting Sockets in the **Inspector** panel (the value of Sockets represents the number of attachment slots).
+
+    ![attach2](./spine/attach2.png)
+
+4. Set the Path and Target of Sockets, select the location and child node object you want to attach respectively.
+
+    ![attach3](./spine/attach3.png)
+
+    You can see that a Sprite has been attached on the tail of the dragon in the **Scene** panel, and the same for the other attachment slots.
+
+    The same goes for the other attachment slots.
+    
+    ![attach4](./spine/attach4.png)
+
+5. Finally, drag the star asset to the Sprite component's `SpriteFrame` property. Save the scene and click the preview button at the top of the editor to see the star hanging on the dragon's tail and wiggling along with it. See the [SpineAttach](https://github.com/cocos-creator/test-cases-3d/tree/v3.0/assets/cases/spine) example in the example-case for details.
+
+## Spine Collision Detection
+
+The Spine attachment slots function allows you to detect the collision of a part of the skeleton animation. The following is an example of how Spine implements collision detection, by determining whether the character's feet are in contact with the ground or not to dynamically change the ground color when the character is running.
+
+![collider](./spine/collider0.png)
+
+1. First, you need to set the 2D physics engine to **Built-In Physics Engine** in the project settings.
+
+    ![collider](./spine/collider1.png)
+
+2. Create the Spine node and its attached child nodes as in the first two steps of the Spine attachment slots via the editor.
+
+3. Then select the target skeleton node (the character's foot) as the parent node in the skeleton node tree under the **Hierarchy** panel Spine node, and create an empty node (renamed frontFoot) as the child node.
+
+    ![collider](./spine/collider2.png)
+
+4. Select the node you want to set (renamed frontFoot) in the **Hierarchy** panel, click **Add Component -> Physics2D -> Colliders -> Polygon Collider** in the **Properties Inspector**, and set the collision component parameters. The node will then move along with the skeletal animation, so that the collision component's enclosing box will be synchronized with the skeletal animation in real time.
+
+    ![collider](./spine/collider3.png)
+
+    ![collider](./spine/collider4.png)
+
+5. Create a Sprite node as a ground in the **Hierarchy** panel. Select the node, then set the position size and other properties in the **Inspector** panel and add the **BoxCollider2D** collision component.
```

Comment:
as a ground

what does this mean? A node that represents a piece of the ground in the scene?

Reply:
Yes. Should I correct it to "as the ground"?

Reply:
Sure, that is a good fix.